airframe-sql/src/main/scala/wvlet/airframe/sql/model/LogicalPlan.scala

```diff
@@ -795,8 +825,10 @@ object LogicalPlan {
     override def inputAttributes: Seq[Attribute] = Seq.empty // TODO
     override def outputAttributes: Seq[Attribute] = {
       columns.map {
-        case a: ArrayConstructor => SingleColumn(MultiColumn(a.values, None), None, None, a.nodeLocation)
-        case other               => SingleColumn(other, None, None, other.nodeLocation)
+        case _: ArrayConstructor =>
+          ResolvedAttribute(UUID.randomUUID().toString, DataType.AnyType, None, Nil, None) // TODO data type
```

Comment:
Removed holding all values in outputAttributes of `Unnest`, but `dataType` cannot be resolved.